feature_appview/src/main/java/cm/aptoide/pt/feature_appview/data/network/service/AppViewNetworkService.kt

```diff
@@ -0,0 +1,23 @@
+package cm.aptoide.pt.feature_appview.data.network.service
+
+import cm.aptoide.pt.aptoide_network.data.network.base_response.BaseV7DataListResponse
+import cm.aptoide.pt.feature_appview.data.network.RemoteAppViewRepository
+import cm.aptoide.pt.feature_appview.data.network.model.RelatedCardJson
+import retrofit2.Response
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+class AppViewNetworkService(private val appViewRemoteDataSource: Retrofit) :
+  RemoteAppViewRepository {
+
+  override suspend fun getRelatedContent(packageName: String): Response<BaseV7DataListResponse<RelatedCardJson>> {
+    return appViewRemoteDataSource.getRelatedContent(packageName)
+  }
+
+  interface Retrofit {
+    @GET("user/action/item/cards/get/type=CURATION_1/aptoide_uid=0/limit=10/")
+    suspend fun getRelatedContent(
```

Comment:
related content needs a package_name, it should only be related to the App in the App View

Reply:
It has package name, it is in the line below as query